test/flows/maternity_paternity_pay_leave_flow_test.rb

```diff
@@ -177,6 +202,7 @@ class MaternityPaternityPayLeaveFlowTest < ActiveSupport::TestCase
       should "have a next node of partner_started_working_before_continuity_start_date " \
                "for a 'yes' response " \
                "when there are two carers " \
+               "and the partner is from England"  \
```

Comment:
It seems that two spaces have been added between the double-quotes and the backslash, rather than one before the closing double-quote, and one after (applies to subsequent lines as well).